src/adapters/scene.js

```diff
@@ -102,9 +116,30 @@ Scene.prototype.initMapBox = function() {
       this.popup.addListener(interactiveLayer);
     });
 
+    // we have to delay click event resolution to make time for possible double click events,
+    // which are thrown *after* two separate click events are thrown
+    this.clickDelayHandler = null;
     this.mb.on('click', e => {
       const pois = this.mb.queryRenderedFeatures(e.point, { layers: interactiveLayers });
-      this.clickOnMap(e.lngLat, pois[0]);
+      // when clicking on a POI, just trigger the action without delay,
+      // as a subsequent double click isn't a problem
+      if (pois[0]) {
+        this.clickOnMap(e.lngLat, pois[0]);
+        return;
+      }
+      // cancel the previous click handler if it's still pending
+      clearTimeout(this.clickDelayHandler);
+      // if this is a real mouse double-click, we can simply return here
+      if (e.originalEvent.detail >= 2) {
+        return;
+      }
```

Comment:
Maybe this early return should be checked before `if (pois[0])` ?  
Or is there a reason why a second click should trigger a click on a POI ?

Reply:
I think you're right. I'll move things along and it should spare a click.